config/jobs/etcd/etcd-presubmits.yaml

```diff
@@ -25,23 +25,55 @@ presubmits:
             cpu: "4"
             memory: "4Gi"
 
-  - name: pull-etcd-unit-test
+  - name: pull-etcd-unit-test-amd64
     cluster: eks-prow-build-cluster
     always_run: true
     branches:
     - main
     decorate: true
     annotations:
       testgrid-dashboards: sig-etcd-presubmits
-      testgrid-tab-name: pull-etcd-unit-test
+      testgrid-tab-name: pull-etcd-unit-test-amd64
     spec:
       containers:
       - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
         command:
         - runner.sh
         args:
-        - make
-        - test-unit
+        - bash
+        - -c
+        - |
+          export JUNIT_REPORT_DIR=${ARTIFACTS}
+          GOOS=linux GOARCH=amd64 CPU=4 GO_TEST_FLAGS='-p=2' make test-unit
+        resources:
+          requests:
+            cpu: "4"
+            memory: "2Gi"
+          limits:
+            cpu: "4"
+            memory: "2Gi"
+
+  - name: pull-etcd-unit-test-386
+    cluster: eks-prow-build-cluster
+    optional: true # remove this once the job is green
+    always_run: true
+    branches:
+    - main
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-etcd-presubmits
+      testgrid-tab-name: pull-etcd-unit-test-386
+    spec:
+      containers:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20240515-17c6d50e24-master
+        command:
+        - runner.sh
+        args:
+        - bash
+        - -c
+        - |
+          export JUNIT_REPORT_DIR=${ARTIFACTS}
+          GOOS=linux GOARCH=386 CPU=1 GO_TEST_FLAGS='-p=4' make test-unit
```

Comment:
If we are limiting to once CPU here should we also update requests/limits for the pod to 1 CPU?

Reply:
I left it with what the original import to Prow had. Notice that it has `optional: true`, as I'm monitoring these jobs and adjusting resources after some runs. Based on my observations from the integration tests, even when limited to one CPU, it may consume a bit more, especially at the beginning of the job.

Reply:
Refer to: https://monitoring-eks.prow.k8s.io/d/96Q8oOOZk/builds?var-org=etcd-io&var-repo=etcd&var-job=pull-etcd-integration-1-cpu-amd64&orgId=1&refresh=30s&from=now-7d&to=now&var-build=All

I'm fine setting it to one, though, as CPU limits in Kubernetes are soft.

Reply:
All good, let's leave it as is and tune later as needed :)